spec/models/photo_spec.rb

```diff
@@ -141,36 +141,43 @@ def with_carrierwave_processing(&block)
   end
 
   context 'with a saved photo containing EXIF data' do
-    before do
-      @exif_filename  = 'exif.jpg'
-      @exif_name      = File.join(File.dirname(__FILE__), '..', 'fixtures', @exif_filename)
-    end
 
-    it 'should contain EXIF data if user prefer' do
-      @alice_photo  = alice.build_post(:photo, :user_file => File.open(@exif_name), :to => alice.aspects.first.id)
+    let (:base_path)       { File.dirname(__FILE__) }
+    let (:public_path)     { File.join(base_path, '../../public/') }
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.